main.go

```diff
@@ -28,7 +30,19 @@ var (
 	}
 )
 
+// Choose which KMS Key to use for signing
+// The default is to use the env var HALLOW_KMS_KEY_ARN, but you could
+// customize this based on the API Gateway request event.
+func ChooseSigner(event events.APIGatewayProxyRequest) (crypto.Signer, error) {
```

Comment:
I'd prefer this named something like `hallow.DefaultSigner` or something more explicit, like `hallow.KMSEnvSigner`, but that's not a huge one.

Reply:
Alternatively this could be a function that allows passthrough, but again, this isn't a suggestion, just floating ideas. 

```
func PassthroughSigner(s crypto.Signer) func(events.APIGatewayProxyRequest) (crypto.Signer, error) {
    return func(events.APIGatewayProxyRequest) (crypto.Signer, error) {
        return s, nil
    }
}
```